tools/src/main/java/org/apache/kafka/trogdor/coordinator/CoordinatorClient.java

```diff
@@ -244,7 +246,7 @@ public static void main(String[] args) throws Exception {
         } else if (res.getBoolean("show_tasks")) {
             System.out.println("Got coordinator tasks: " +
                 JsonUtil.toPrettyJsonString(client.tasks(
-                    new TasksRequest(null, 0, 0, 0, 0))));
+                    new TasksRequest(null, 0, 0, 0, 0, Optional.empty()))));
```

Comment:
Can you file a follow-on JIRA for adding support for specifying the task state to `show_tasks`?

Then the user could run something like:
```
./bin/trogdor.sh client --show-tasks --state DONE
```

Alternately, we could add support in this pull request if you want.

Reply:
I decided we should open a JIRA since it would be good if we supported every argument for --show-tasks (taskids, start/end ms)

Reply:
https://issues.apache.org/jira/browse/KAFKA-7675